lib/util/git/index.ts

```diff
@@ -483,15 +483,20 @@ export async function isBranchModified(branchName: string): Promise<boolean> {
     return false;
   }
   // Retrieve the author of the most recent commit
-  const lastAuthor = (
-    await git.raw([
-      'log',
-      '-1',
-      '--pretty=format:%ae',
-      `origin/${branchName}`,
-      '--',
-    ])
-  ).trim();
+  let lastAuthor: string;
+  try {
+    lastAuthor = (
+      await git.raw([
+        'log',
+        '-1',
+        '--pretty=format:%ae',
+        `origin/${branchName}`,
+        '--',
+      ])
+    ).trim();
+  } catch (err) /* istanbul ignore next */ {
+    logger.warn({ err }, 'Error checking last author for isBranchModified');
+  }
```

Comment:
I found this happening sometimes in the app logs. In such a case I think it's OK to keep going with a warning but assume the branch is modified and it hopefully recovers the next run.